Xamarin.Forms.Core/Binding.cs

```diff
@@ -276,11 +276,7 @@ bool ElementFitsAncestorTypeAndLevel(Element element, ref int level, ref object
 
 		internal override BindingBase Clone()
 		{
-			return new Binding(Path, Mode) {
-				Converter = Converter,
-				ConverterParameter = ConverterParameter,
-				StringFormat = StringFormat,
-				Source = Source,
+			return new Binding(Path, Mode, Converter, ConverterParameter, StringFormat, Source) {
```

Comment:
this is microscopic optimization :P I wonder how it shows up in the measures

Reply:
I did discover that BenchmarkDotNet was messing with me, sometimes it shows `us` (microseconds) and sometimes `ns` (nanoseconds).

No idea why it sometimes prints data like this instead:

```
         Method |     Mean |     Error |    StdDev |   Median | Gen 0/1k Op | Gen 1/1k Op | Gen 2/1k Op | Allocated Memory/Op |
--------------- |---------:|----------:|----------:|---------:|------------:|------------:|------------:|--------------------:|
   ApplySingle2 | 1.527 us | 0.0110 us | 0.0103 us | 1.529 us |      0.0992 |           - |           - |               521 B |
   ApplySingle1 | 1.545 us | 0.0079 us | 0.0074 us | 1.544 us |      0.1087 |           - |           - |               573 B |

// * Legends *
...
  1 us                : 1 Microsecond (0.000001 s
```

I think it was actually saving 1 microsecond per-binding on my Mac, which makes way more sense. I was able to see the total ~3ms improvement on 100 Labels w/ 1 binding on Android, however. 100 bindings might be reasonable on startup for a real app?

Reply:
I removed the change that broke things, and removed an `IEnumerable` / `yield return` instead. Still a ~3.7ms improvement on an app with 100 `Label` with a binding.

I am hoping this passes CI this time. 👍